sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3102,6 +3102,12 @@ object GpuOverrides extends Logging {
                   s" ${RapidsConf.ENABLE_FLOAT_FORMAT_NUMBER} to true.")
               }
             }
+            case dt: DecimalType => {
```

Comment:
Once https://github.com/rapidsai/cudf/issues/14210 is fixed we should come back and retest to be sure that it is working properly.